akka-projection-slick/src/test/scala/akka/projection/slick/SlickOffsetStoreSpec.scala

```diff
@@ -39,7 +37,7 @@ object SlickOffsetStoreSpec {
 
       # TODO: configure connection pool and slick async executor
       db = {
-       url = "jdbc:h2:mem:test1"
+       url = "jdbc:h2:mem:offset-store-test;DB_CLOSE_DELAY=-1"
```

Comment:
it turns out that we were getting errors and not seeing because of `Await.ready`. One of the tests were creating the offset-store table and the other was failing to create it. When changed to `Await.result` it started to fail.

I'm taking the opportunity to have distinct db instance to isolate them.